third_party/airflow/armada/operators/armada.py

```diff
@@ -63,7 +64,7 @@ def __init__(
         armada_client: ArmadaClient,
         job_service_client: JobServiceClient,
         armada_queue: str,
-        job_request_items,
+        job_request_items: armada_client_proto.PodSpec,
```

Comment:
This is not correct.
https://github.com/armadaproject/armada/blob/master/client/python/armada_client/client.py#L113

@Sharpz7 what is this type now?  We never added it to the client but I have a comment saying it is a List[JobRequestItems].

Reply:
Addressed in f1687c7